dataeng/jobs/analytics/LoadVerticaSchemaToBigquery.groovy

```diff
@@ -0,0 +1,32 @@
+package analytics
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_multiscm
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_parameters
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_log_rotator
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_wrappers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_publishers
+import static org.edx.jenkins.dsl.AnalyticsConstants.common_triggers
```

Comment:
Is this being used ?

Reply:
Ah, good catch.  I'll remove that in my next DSL PR.   (I need to make all of this be in turn triggered by SQL scripts for the next story.)